src/overcooked_demo/server/config.json

```diff
@@ -0,0 +1,36 @@
+{
+    "logfile" : "app.log",
+    "layouts" : ["cramped_room", "cramped_room_tomato", "asymmetric_advantages", "coordination_ring", "forced_coordination", "counter_circuit", "cramped_corridor", "marshmallow_experiment", "long_cook_time", "forced_coordination_tomato", "asymmetric_advantages_tomato", "marshmallow_experiment_coordination", "pipeline", "you_shall_not_pass", "tutorial_3"],
+    "MAX_GAMES" : 10,
+    "MAX_GAME_LENGTH" : 120,
+    "AGENT_DIR" : "./static/assets/agents",
+    "MAX_FPS" : 30,
+    "psiturk" : {
```

Comment:
Do you have a sense of where this is used elsewhere in the demo code? This seems like it is related to collecting user data, but how does one turn that option on? We should have some documentation for this if it doesn't exist already